nixos/modules/services/web-apps/kratos.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.kratos;
+
+  upstreamKratosVersion = lib.getVersion pkgs.kratos;
+
+  kratosPackage = if config.services.kratos.enable
+    then
+      config.services.kratos.package
+    else
+      pkgs.kratos;
+
+  kratosVersion = lib.getVersion kratosPackage;
+in {
+  options = {
+    services.kratos = {
+      enable = lib.mkEnableOption "Kratos, Libre and open source identity server";
+
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = pkgs.kratos;
+        defaultText = "pkgs.kratos";
+        description = "The kratos package to use.";
+      };
+    };
+  };
+
+  # config = lib.mkIf cfg.enable {
+  # };
```

Comment:
I am not sure, but I was not able to find the implementation.

Reply:
elaborate "I was not able to find the implementation.". I though that the current is enough to deploy kratos through `services.kratos.enable` ?

Reply:
Sorry I am not familiar with this package, can you briefly describe how we should deploy this without this option? For example, adding the package to `environment.systemPackages` and maybe `systemd.packages`?